config/default.json

```diff
@@ -12,6 +13,15 @@
       ]
     }
   },
+  "SHARED_CACHE": {
```

Comment:
In quotes PR this parameter name is PAYLOAD_CACHE.
https://github.com/mojaloop/quoting-service/pull/357

And also please the other new parameter ORIGINAL_PAYLOAD_STORAGE